lib/siilar/client/search.rb

```diff
@@ -5,7 +5,7 @@ module Search
       # Search for a track
       #
       # @see http://api.siilar.com/1.0/doc/search-and-analyze#search
-      def search(query = {})
+      def similar(query = {})
         options = { query: query }
         response = client.get('1.0/search', options)
```

Comment:
Why not hardcoding `similar` option ?
This would give us smth like : 
```ruby
def similar(ids)
  response = client.get('1.0/search', { similar_tracks: ids })
end
```